study-builder/studies/pancan/snippets/common/cancer-picklist-groups-additional.conf

```diff
@@ -0,0 +1,323 @@
+{
+  "stableId": "CANCER_GROUP_ADDITIONAL_ADDITIONAL_CANCERS",
+  "nameTemplate": {
+    "templateType": "HTML",
+    "templateText": "$prompt",
+    "variables": [
+      {
+        "name": "prompt",
+        "translations": [
+          { "language": "en", "text": ${i18n.en.cancer.cancer_group_additional_additional_cancers} },
+          { "language": "es", "text": ${i18n.es.cancer.cancer_group_additional_additional_cancers} },
+        ]
+      }
+    ]
+  }
+  "options": [
+
+    {
+      "stableId": "CANCER_ADDITIONAL_DF_DESMOID_FIBROSIS",
```

Comment:
Can we get rid of the `CANCER_` prefix? These are used in the cancer questions, so we know they are cancers... The option stable ids are already long enough.

Reply:
May be to shorten `CANCER_` to `C_` (just to visually see in PEX expressions that it is a cancer option:
`answers.hasOptionStartsWith("C_SARCOMAS_OSTEO", "C_BRAIN_")`
Because if we remove the prefix completely then seems it a bit more difficult to read such PEX expressions (difficult to visually find cancer options and non-cancer):
For example without prefix:
```
      (user.studies["cmi-pancan"].forms["PREQUAL"].questions["PRIMARY_CANCER_LIST_SELF"].children["PRIMARY_CANCER_SELF"].answers.hasOption("GASTRO_CRC_COLORECTAL")
      || user.studies["cmi-pancan"].forms["PREQUAL"].questions["PRIMARY_CANCER_LIST_CHILD"].children["PRIMARY_CANCER_CHILD"].answers.hasOption("GASTRO_CRC_COLORECTAL"))
```
With prefix:
```
      (user.studies["cmi-pancan"].forms["PREQUAL"].questions["PRIMARY_CANCER_LIST_SELF"].children["PRIMARY_CANCER_SELF"].answers.hasOption("C_GASTRO_CRC_COLORECTAL")
      || user.studies["cmi-pancan"].forms["PREQUAL"].questions["PRIMARY_CANCER_LIST_CHILD"].children["PRIMARY_CANCER_CHILD"].answers.hasOption("C_GASTRO_CRC_COLORECTAL"))
```

Reply:
I replaced "CANCER_" with "C_". Now the ids became shorter.

Reply:
I still recommend removing `C_`. In my view, it's not about if it looks good to us or if it is readable to us. It's about study staff. Study staff will see these option stable ids in the data. Let's make these names easier for them.

Reply:
Will remove (didn't know about study staff usage).